core/modules/layout/css/layout.admin.css

```diff
@@ -374,3 +374,19 @@
   margin-right: 2em;
   display: inline;
 }
+
+.edit-layout-top {
+  padding: 0.5em 0.5em;
+  background: #e9e8e8;
+  -webkit-border-radius: 4px 4px 0 0;
+  -moz-border-radius: 4px 4px 0;
+  border-radius: 4px 4px 0 0;
```

Comment:
We only use `border-radius` everwhere else in this file, I think the -webkit and -moz ones can be deleted.